src/core/public/i18n/__snapshots__/i18n_service.test.tsx.snap

```diff
@@ -54,14 +53,21 @@ exports[`#start() returns \`Context\` component 1`] = `
           "euiStepNumber.hasErrors": "has errors",
           "euiStepNumber.hasWarnings": "has warnings",
           "euiStepNumber.isComplete": "complete",
+          "euiSuperDatePicker.showDatesButtonLabel": "Show dates",
           "euiSuperSelect.screenReaderAnnouncement": [Function],
           "euiSuperSelectControl.selectAnOption": [Function],
+          "euiSuperUpdateButton.cannotUpdateTooltip": "Cannot update",
```

Comment:
@elastic/kibana-platform do we need this test? we pass create object and pass it to Context as is.